src/services/VoteService.ts

```diff
@@ -36,4 +46,45 @@ export class VoteService {
       .map((option) => option.identifier)
       .includes(vote);
   }
+
+  public async countVotes(
+    ballotID: string
+  ): Promise<TotalVoteCount> {
+    const votes = await this.getVotes(ballotID);
+    return {
+      count: votes.length
+    };
+  }
+
+  public async getVoteResult(
+    ballotID: string,
+    voteOptions: VotingOption[]
```

Comment:
Don't see a reason why `ballotID` wouldn't be enough as you can query the voteOptions using the id directly.

Reply:
ok